### DISCUSSION TITLE
Stitches syntax support

### DISCUSSION THREAD

**Discussion by alanontheweb** [ben-rogerson/twin.macro#357] [Ideas]
Hi @ben-rogerson , I was looking over this new project that aims to combine TWind with a Stitches inspired API - https://github.com/tw-in-js/twind-react

Any chance this support could come to twin.macro? Seems like a clean way to handle variants.

**Comment by ben-rogerson** [comment]
Hey Alan

I've got a couple on the list already like Linaria + Compiled but I'll add Stitches to the list.
There's quite a bit of work that goes into adding support for new libraries so I can't promise full support will be added soon.

🔥  I’ve added an experimental example of [stitches running with twin](https://github.com/ben-rogerson/twin.examples/tree/master/next-stitches-typescript)

**Reply by alanontheweb** [reply]
Ooh amazing! I'll check it out :)
